bot/code_review_bot/cli.py

```diff
@@ -176,6 +179,19 @@ def main():
         # Update Harbormaster status
         w.update_status(revision, state=BuildState.Fail)
 
+        failure = UnitResult(
```

Comment:
Could you report this to lando too, like we do in events?

Reply:
Sure thing, same error as a generic failure.